include/zephyr/net/zperf.h

```diff
@@ -51,10 +51,10 @@ struct zperf_results {
 	uint32_t nb_packets_rcvd;
```

Comment:
What does `TP` in commit message mean?

Reply:
Hi @jukkar 
I will remove the code formatting change.
The TP is throughput, sorry for the confusing, I will change it too.
Actually I think the work is finalized and have tested locally with iperf for both udp and tcp for long time.

Reply:
Hi @jukkar 
please help review again, have updated.